utils/build/build.js

```diff
@@ -116,17 +140,30 @@ onChanges.push({
     'docs/src/api/',
     'utils/generate_types/overrides.d.ts',
     'utils/generate_types/exported.json',
-    'src/server/chromium/protocol.ts',
+    'src/server/chromium/protocol.d.ts',
     'src/trace/traceTypes.ts',
   ],
   script: 'utils/generate_types/index.js',
 });
 
-// Copy images.
-steps.push({
-  command: process.platform === 'win32' ? 'copy' : 'cp',
-  args: [filePath('src/web/recorder/*.png'), filePath('lib/web/recorder/')],
-  shell: true,
+copyFiles.push({
+  files: 'src/web/recorder/*.png',
+  from: 'src',
+  to: 'lib',
+});
+
+copyFiles.push({
+  files: 'src/**/*.js',
```

Comment:
Could you please share the examples here that we want to copy?

Reply:
diff_match_patch.js, deviceDescriptorsSource.json. Added them in a comment.